auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/request_handlers/api/LoginApiRequestHandler.kt

```diff
@@ -55,7 +55,7 @@ class LoginApiRequestHandler : ApiRequestHandler {
             override fun onSuccess(credentials: Credentials) {
                 val scope = credentials.scope?.split(" ") ?: listOf()
                 val sdf =
-                    SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ", Locale.getDefault())
+                    SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ", Locale.US)
```

Comment:
Can we use the format `yyyy-MM-dd'T'HH:mm:ss.SSS'Z'` as that is the one used in Android and iOS.